src/WebApi/UnitTests/Tests/Application/UseCase/QuestionTests/DeleteQuestionUseCaseTest.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Application.Repositories;
+using Application.UseCases.Question.DeleteInterviewQuestion;
+using Domain.Models;
+using Moq;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace UnitTests.Tests.Application.UseCase.QuestionTests
+{
+    public class DeleteQuestionUseCaseTest
+    {
+        private static DeleteQuestionInput Input => new DeleteQuestionInput()
+        {
+            Id= 1
```

Comment:
Just a minor styling issue with a suggestion
```suggestion
            Id = 1
```